api/v1alpha1/meshsync_types.go

```diff
@@ -34,10 +34,39 @@ type MeshsyncBroker struct {
 	Native NativeMeshsyncBroker `json:"native,omitempty" yaml:"native,omitempty"`
 }
 
+// an array of resources that meshsync listes to and publishes their events
+type PipelineConfigs []PipelineConfig
+
+// resources that meshsync observes and publishes to a given subscriber via the broker
+type PipelineConfig struct {
+	Name      string `json:"name" yaml:"name"`
+	PublishTo string `json:"publish-to" yaml:"publish-to"`
```

Comment:
I saw the PR in mesheysync repo. It is hard to say we can keep the same style.

Reply:
Due to the varied nature of both repos, it can only be consistent on one side. Given that situation, would you advice that status quo remains or the changes be consistent with the above shared example?

Reply:
To be honest, I am not sure which style we should follow but these days I have checked the code from other repo. It looks like "publish-to" was used in there too. So, I believe the format in this PR is ok. And this is the reason for above I mentioned "it is hard to say we can keep the same style."